boofuzz/blocks/size.py

```diff
@@ -220,7 +220,10 @@ def __repr__(self):
         return "<%s %s>" % (self.__class__.__name__, self._name)
 
     def __len__(self):
-        return self.length
+        if self.format == "ascii":
+            return len(str(self._calculated_length()))
+        else:
+            return self.length
```

Comment:
Spring-boarding off of @SR4ven 's suggestion -- Would this work too? If so, it would avoid some code reuse:
```python
    def __len__(self):
        return len(self._render())
```

The code in _render() is very slightly different -- I'm not sure if it's more accurate, but at least the logic would be in one place.

Reply:
Yes, that looks like a good idea. It works just as good in my test cases. I updated the pull request.

Reply:
Good catch @jtpereyda!

The same code seems to work for `bit_field`
```python
    def __len__(self):
        return len(self._render(self._value))
```

If someone else can verify that this still does what it's supposed to, I suggest applying this change.

Reply:
@mshaneck are you able to apply/test that change in bit_field too?

Reply:
Yeah. Looks good. I updated the pull request.